RedHatAI/Mistral-Small-24B-Instruct-2501-FP8-Dynamic/accuracy/client.yml

```diff
@@ -1,4 +1,3 @@
 # llm-eval-test configs for https://huggingface.co/RedHatAI/Mistral-Small-24B-Instruct-2501-FP8-Dynamic
 model: "RedHatAI/Mistral-Small-24B-Instruct-2501-FP8-Dynamic"
-chat_template: true
-fewshot_as_multiturn: true
+no-chat-template: true
```

Comment:
I see on this model's HF page, the evaluation for leaderboard is still setting chat-template to true, the evaluation for openllm is not setting it though.

Maybe we should consider combining client and task configs together so that it can be more accurate.

Reply:
Discussing this in Slack.  It gets more complicated.  some other models have `chat-template` only enabled for certain tasks in the openllm group.